internal/capability/audit.go

```diff
@@ -84,7 +84,7 @@ func newCapAudit(ctx context.Context, c operator.Client, subscription operator.S
 		namespace:         ns,
 		operatorGroupData: newOperatorGroupData(operatorGroupName, getTargetNamespaces(subscription, ns)),
 		subscription:      subscription,
-		csvWaitTime:       time.Minute,
+		csvWaitTime:       2 * time.Minute,
```

Comment:
Is this 2x of time specified explicitly known to the consumer of this param ? i.e. why doesnt the consumer explicitly set the appropriate timeout instead of its being "automagically" 2*ed ?

Reply:
That value is 2 minutes. time.Minute means 1 minute. The idea is to increase the time we give for CSVs to timeout. That parameter is where the "consumer" changes it. Unless we come up with something like a flag to set it from the CLI but that's a feature request.